--- origsrc/pspp-0.6.2/lib/gtksheet/gtkitementry.c	2009-09-17 22:05:40.000000000 -0500
+++ src/pspp-0.6.2/lib/gtksheet/gtkitementry.c	2011-01-25 05:21:26.473354000 -0600
@@ -34,6 +34,10 @@
 #include <gtk/gtk.h>
 #include "gtkitementry.h"
 
+#if GTK_CHECK_VERSION (2,22,0)
+#define gtk_entry_reset_im_context pspp_entry_reset_im_context
+#endif
+
 #if GTK_CHECK_VERSION (2,17,4)
 #define HAVE_GTKENTRYBUFFER 1
 #endif
