--- origsrc/pspp-0.8.1/perl-module/Makefile.PL	2013-04-28 11:46:44.000000000 -0500
+++ src/pspp-0.8.1/perl-module/Makefile.PL	2013-12-10 00:57:49.093147000 -0600
@@ -22,7 +22,7 @@ WriteMakefile(
       (ABSTRACT       => 'Perl extension to PSPP',
        AUTHOR         => 'John Darrington <john@darrington.wattle.id.au>') : ()),
     INC       => "-I $Locations{SourceDir}  -I $Locations{SourceDir}/src -I $Locations{SourceDir}/gl -I $Locations{BuildDir}/gl -I $Locations{BuildDir}", 
-    MYEXTLIB  => "$Locations{BuildDir}/src/.libs/libpspp-core.\$(SO)",
+    MYEXTLIB  => "$Locations{BuildDir}/src/.libs/libpspp-core.dll.a",
     MAN3PODS  => {"lib/PSPP.pm", "\$(INST_MAN3DIR)/PSPP.3pm",
 	    "Examples.pod", "\$(INST_MAN3DIR)/PSPP::Examples.3pm"}
 );
